src/cloudcasting/validation.py

```diff
@@ -183,13 +196,38 @@ def score_model_on_all_metrics(
         drop_last=False,
     )
 
-    metric_funcs: dict[str, Callable[[BatchOutputArray, BatchOutputArray], MetricArray]] = {
-        "mae": mae_batch,
-        "mse": mse_batch,
-        # "ssim": ssim_batch,  # currently unstable with nans
-    }
+    if metric_kwargs is None:
+        metric_kwargs_dict: dict[str, dict[str, Any]] = {}
+    else:
+        metric_kwargs_dict = metric_kwargs
+
+    def get_pix_function(
```

Comment:
For documentation purposes: this is a way to fish functions from `dm_pix` by name, which is a little bit over-engineered since we now have the metric code in our library. Will re-visit at a later stage once things are more mature.